lib/l10n/intl_fr.arb

```diff
@@ -31,7 +31,7 @@
   "login_info_universal_code": "Votre code d'accès universel est sous la forme AB12345. Celui-ci vous a été transmis par le Bureau du registraire lors de votre admission.",
   "login_msg_logout_success": "Vous vous êtes déconnecté",
   "login_applets_logo": "Conçue par",
-  "password_forgotten": "Mot de passe oublié?",
+  "login_password_forgotten": "Mot de passe oublié?",
```

Comment:
make sure to make these changes to intl_en.arb also

Reply:
This label is already on intl_en.arb :) the label name was different on intl_fr.arb